packages/mdc-button/README.md

```diff
@@ -40,38 +40,54 @@ npm install --save @material/button
 
 ## Usage
 
-### Flat
+### Button type
 
-```html
-<button class="mdc-button">
-  Flat button
-</button>
-```
+> note: Examples and documents use generic `<button>`, but users can also apply
+> `mdc-button` to `<a class="mdc-button">Link Button</a>` if that makes semantic
+> meaning.
 
-### Colored
+#### Text Button
 
 ```html
-<button class="mdc-button mdc-button--accent">
-  Colored button
+<button class="mdc-button">
+  Text button
 </button>
 ```
 
-### Raised
+#### Raised Button
 
 ```html
 <button class="mdc-button mdc-button--raised">
   Raised button
 </button>
 ```
 
-### Disabled
+### Button state
+
+#### Disabled
+
+Users can add `disabled` directly to button element or set the fieldset containing
+the button to `disabled` to disable a button. Disabled button could not be interacted
+with and have no visual interaction effect.
 
 ```html
 <button class="mdc-button mdc-button--raised" disabled>
   Raised disabled button
 </button>
 ```
 
+### Colored
+
+Button have a default baseline color of rgb(98, 25, 130), but it could also be set
+to adopt application's primary and accent color by adding `mdc-button--primary` or
```

Comment:
* "adopt application's" -> "adopt the application's"
* "by adding ..." -> "by adding the ..."